Sources/UI/Extensions/UIKit/UIEdgeInsets+Extensions.swift

```diff
@@ -24,4 +24,11 @@ extension UIEdgeInsets {
     public static func all(_ value: CGFloat) -> UIEdgeInsets {
         return UIEdgeInsets(all: value)
     }
+    
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(top)
+        hasher.combine(left)
+        hasher.combine(bottom)
+        hasher.combine(right)
+    }
```

Comment:
Isn't this default implementation?

Reply:
You're right!

Reply:
but `Implementation of 'Hashable' cannot be automatically synthesized in an extension in a different file to the type `

Reply:
Interesting, why do we need `Hashable` in this type?

Reply:
To make `MessageViewStyle` hashable.